### PR TITLE
Cleaning up/fixing a lot of bad "send to world" code

### LINKED ISSUE #14133: Darkspawn mindlink test in lobby
### Severity

Low

### Short Description

Darkspawn "X has not divulged in time and is now forcefully divulging" shows up in main menu

### Reproduction

Wait ~3 days for a darkspawn round in the main menu

### Round ID

40969

### Test Merges

#13684 #14061 #13954 #14085

### BYOND Username

GoreDem

### Additional information

gruh

also gamma alert can be heard in the menu if nukies war

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Well this started out as a simple bugfix and I realized this was all over the damn place.

A lot of places used `to_chat(world` which sends a message to ALL CONNECTED PLAYERS including ones in the lobby. This has been substituted with the proper `send_to_playing_players`.

Other places would SEND_SOUND to all players, and this was changed to the proper `sound_to_playing_players`.

Along the way I found some debug messages that were left in `to_chat(world` so in the off chance that something went wrong and they actually PLAYED, all players would get `lover != Brutus` or `uh oh, stinky!!` in their chat. Very cool.

Some places were intentionally left out, particularly the Blob and Malf victory messages. As this ends the round instantly, and Jamie says not to touch it, it seems reasonable to me to keep it there and let lobby players know what happened. Until a maintainer says that shitting up the lobby chat is a bad idea, of course.
`to_chat(world, "<B>[real_name] consumed the station in an unstoppable tide!</B>")`
`to_chat(world, "<B>The AI cleansed the station of life with the doomsday device!</B>")`

There's a lot of other debug stuff in here but it seems pretty technical like savefile stuff so I'm not going to touch it until someone says to.

Closes #14133

# Changelog

:cl:  
bugfix: Fixed darkspawn force divulge being shown to lobby players
bugfix: Fixed darkspawn sacrament noise being heard by lobby players
bugfix: Fixed shadowling ascendant death being shown to lobby players
bugfix: Fixed shadowling ascend message being shown to lobby players
bugfix: Fixed shadowling ascend noise being heard by lobby players
bugfix: Fixed a particular cult god fight being shown to lobby players
bugfix: Fixed Gamma & Epsilon alerts being heard by lobby players
bugfix: Fixed Epsilon alert giving lobby players a bad feeling
/:cl:

### CODE REVIEW

File: yogstation/code/modules/antagonists/darkspawn/darkspawn.dm [hunk 1]
```diff
@@ -334,11 +334,9 @@
 	H.do_jitter_animation(1000)
 	var/processed_message = span_velvet("<b>\[Mindlink\] [H.real_name] has not divulged in time and is now forcefully divulging.</b>")
 	for(var/mob/M in GLOB.player_list)
-		if(M.stat == DEAD)
-			var/link = FOLLOW_LINK(M, H)
-			to_chat(M, "[link] [processed_message]")
-		else if(isdarkspawn(M))
+		if(M.stat != DEAD && isdarkspawn(M))
```
Comment (Bibby0110) [inline yogstation/code/modules/antagonists/darkspawn/darkspawn.dm:337]: won't you get double messages this way?

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Comment by bruhlookatthisdood** [comment]
will this change people off the Z level getting station announcments?
like the old crew hearing about the shuttle, or announcments, or nar nar rising up?

**Comment by ynot01** [comment]
> will this change people off the Z level getting station announcements

No

**Review by JamieD1 — APPROVED** [review]

File: yogstation/code/modules/antagonists/darkspawn/darkspawn.dm (hunk 1 above)
Comment (ynot01) [reply yogstation/code/modules/antagonists/darkspawn/darkspawn.dm:337]: no

**Pull request merged**